### PR TITLE
Student Profile Page

### PR DESCRIPTION
**Pull request by serenity2423**
-can edit profile details
-can fetch user details
-can update user details in the database
-added dropdown button for the password
-when update checked, the input values cleared

### CODE REVIEW

File: frontend/src/Components/Profile/ProfilePage.css [hunk 1]
```diff
@@ -192,7 +192,16 @@ h1{
     background: rgb(0, 105, 217);
 }
 
-
+.dropdown-toggle {
```
Comment (razstvien01) [inline frontend/src/Components/Profile/ProfilePage.css:195]: WAAWWW

File: frontend/src/Components/Profile/ProfilePage.jsx [hunk 2]
```diff
@@ -148,31 +149,41 @@ export const ProfilePage = () => {
                 {/* // Disable input if not in edit mode */}
               </input>
             </div>
-            <div className="col-5">
```
Comment (razstvien01) [inline frontend/src/Components/Profile/ProfilePage.jsx:151]: ilisii ni

File: frontend/src/Components/Profile/ProfilePage.jsx [hunk 3]
```diff
@@ -148,31 +149,41 @@ export const ProfilePage = () => {
...
-                disabled={!editMode}
+            <div>
+              <button
+                className="dropdown-toggle"
+                type = "button"
+                onClick={() => setShowDropdown(!showDropdown)}
               >
-                {/* // Disable input if not in edit mode */}
-              </input>
-              <input
```
Comment (razstvien01) [inline frontend/src/Components/Profile/ProfilePage.jsx:166]: change this too blahc blah blah

File: frontend/src/Components/Profile/ProfilePage.jsx [hunk 4]
```diff
@@ -148,31 +149,41 @@ export const ProfilePage = () => {
...
+                      id="password"
+                      placeholder="Enter password"
+                      value={formData.password}
+                      onChange={handleInputChange}
+                      disabled={!editMode}
+                    />
+                    <input
+                      type="password"
+                      id="confirmPassword"
+                      placeholder="Enter confirm password"
```
Comment (razstvien01) [inline frontend/src/Components/Profile/ProfilePage.jsx:179]: Goods

Comment (razstvien01) [reply frontend/src/Components/Profile/ProfilePage.jsx:179]: DONE\

File: frontend/src/Components/Profile/ProfilePage.jsx (hunk 3 above)
Comment (serenity2423) [reply frontend/src/Components/Profile/ProfilePage.jsx:166]: done.

**Pull request merged**